2-Try-Trainer-class_en.ipynb

```diff
@@ -104,19 +104,20 @@
     "        h2 = F.relu(self.l2(h1))\n",
     "        return self.l3(h2)\n",
     "\n",
-    "gpu_id = 0\n",
+    "gpu_id = 0  # Set to -1 if you don't have a GPU\n",
     "\n",
     "model = MLP()\n",
-    "model.to_gpu(gpu_id)"
+    "if gpu_id >= 0:\n",
+    "        model.to_gpu(gpu_id)"
```

Comment:
Could you use 4 spaces as 1 tab? There are 8 spaces before `model.to_gpu(gpu_id)`. 4 spaces are better.